packages/block-library/src/search/block.json

```
@@ -83,6 +91,7 @@
 		},
 		"html": false
 	},
+	"viewScript": "file:./view.js",
```

```suggestion
	"viewScript": "file:./view.min.js",
```

Should this be the `.min` version of the script? I see this is the version used in the [file block](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/file/block.json#L70) and the [navigation block](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/navigation/block.json#L137).

I'm also seeing a PHP notice on this branch, which the `.min` version of this script seems to fix:

```
Notice: Function register_block_script_handle was called <strong>incorrectly</strong>. The asset file (/build/block-library/blocks/search/view.asset.php) for the "viewScript" defined in "core/search" block definition is missing.
```

---

Should be resolved by [73176c1](https://github.com/WordPress/gutenberg/pull/50487/commits/73176c118e8098413b5880557930b4a57e1446a6).